src/changelog-fetcher.js

```diff
@@ -35,7 +35,7 @@ class ChangelogFetcher {
    */
 
   assignPullRequestToRelease(releases, pr) {
-    const release = find(releases, release => pr.merged_at.isSameOrBefore(release.created_at));
+    const release = _.find(releases, release => pr.merged_at.isSameOrBefore(release.created_at));
```

Comment:
Maybe we can replace `_.find()` with the native alternative?

```suggestion
    const release = releases.find(release => pr.merged_at.isSameOrBefore(release.created_at));
```

Reply:
Sure